translate/lang/test_scn.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from translate.filters.test_checks import fails, passes
+from translate.lang.ro import SicilianChecker
```

Comment:
Fix this import.

Reply:
Done in https://github.com/translate/translate/pull/3919/commits/fc701bc9f4c4187b77250373a630bc6aba71d13b , sorry for the mistake (yes, I used the Romanian-language checks as a reference :) ).

Thank you!
David